menu.go

```diff
@@ -73,8 +76,30 @@ func buildMenu(c Commands, m Module, opts *buildMenuOptions) (menu, []error) {
 
 	a := argsRelativeTo(m, nil)
 	helpUsage := strings.Join(append([]string{a[0], "help"}, a[1:]...), " ")
+	usage := Usage(m) + " <command> [<args>]"
+
+	return &menu{Usage: usage, Sections: sections, HelpUsage: helpUsage}, errs
+}
+
+func expandModules(cmds Commands, depth int) (Commands, []error) {
+	all := Commands{}
+	var errs []error
+
+	for _, cmd := range cmds {
+		if m, ok := cmd.(Module); ok && depth != 0 {
+			subcmds, err := m.Subcommands()
+			if err != nil {
+				errs = append(errs, err)
+			}
+			all2, errs2 := expandModules(subcmds, depth-1)
```

Comment:
nit: Are there better names here than `all2` and `errs2` ?

Reply:
I renamed them to `expandedSubcmds` and `expansionErrs`, respectively.